pyproject.toml

```diff
@@ -41,6 +41,8 @@ dependencies = [
     "fiona>=1.8.22",
     # geopandas 0.12.1+ required for Shapely 2+ support
     "geopandas>=0.12.1",
+    # gcsfs 2023.10.0+ required for Google Cloud Storage support
```

Comment:
Moves this to the "models" section below as an optional dependency.

Also need to add neuralgcm to the same section.